Bottom: 454080d2f181264e1feec7efa84a91b04ec45a2f
Top:    6869788182028a94bb620f43520b4eb342a4f7bf
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 18:03:37 +0000

Refresh of attempt-to-refactor-the-step

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 9b91a07..f4688ac 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -15,7 +15,7 @@
 #include "gui-pinout-preview.h"
 #include "pcb-printf.h"
 
-#include "hid/step/step_writer.h"
+#include "hid/common/step_id.h"
 #include "hid/common/quad.h"
 #include "hid/common/vertex3d.h"
 #include "hid/common/contour3d.h"
